client/state/concierge/actions.js

```diff
@@ -8,11 +8,24 @@ import {
 	CONCIERGE_AVAILABLE_TIMES_UPDATE,
 	CONCIERGE_APPOINTMENT_CANCEL,
 	CONCIERGE_APPOINTMENT_CREATE,
+	CONCIERGE_APPOINTMENT_DETAILS_REQUEST,
+	CONCIERGE_APPOINTMENT_DETAILS_UPDATE,
 	CONCIERGE_APPOINTMENT_RESCHEDULE,
 	CONCIERGE_SIGNUP_FORM_UPDATE,
 	CONCIERGE_UPDATE_BOOKING_STATUS,
 } from 'state/action-types';
 
+export const requestConciergeAppointmentDetails = ( scheduleId, appointmentId ) => ( {
```

Comment:
Theres probably plans to use this action in the future since theres some tests but I didn't see it used in any of the code in this PR. Just bringing it up just incase it was just left over

Reply:
Your assumption was correct, this will be used in another PR (https://github.com/Automattic/wp-calypso/pull/21732), this intend is stated in the description also `Also creates support for requesting concierge appointment details which will be used by the cancel and reschedule flows.`